### PR TITLE
Fix: Update lerna and fix fs.unlink call

### PR DESCRIPTION
**Pull request by ikovac**
This PR:
- fixes development workflow by updating lerna version
- fixes a few minor bugs such as fs.unlink call
- sort package.json

### CODE REVIEW

**Review by rkusan00 — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -42,6 +42,9 @@ jobs:
       - run:
           name: Setup tailor config
           command: cp ./tailor.config.js.example ./tailor.config.js
+      - run:
+          name: Build core packages
+          command: npm run packages:build
```
Comment (underscope) [inline .circleci/config.yml:47]: Can you address the CI build step failure?

File: package.json [hunk 2]
```diff
@@ -1,41 +1,40 @@
 {
   "name": "tailor",
   "version": "5.0.0-rc.5",
-  "codename": "twill",
+  "private": true,
```
Comment (underscope) [inline package.json:4]: Was there a specific reason to introduce these changes?

File: packages/core-components/src/components/ContentPreview/index.vue [hunk 3]
```diff
@@ -44,7 +45,7 @@ export default {
       if (!allowedTypes.length) return containers;
       return containers.map(container => ({
         ...container,
-        elements: container.elements.filter(it => allowedTypes.includes(it.type))
+        elements: intersectionBy(container.elements, allowedTypes, 'type')
```
Comment (underscope) [inline packages/core-components/src/components/ContentPreview/index.vue:48]: How did this one ended up here 🤔

**Review by underscope — APPROVED** [review]

File: packages/core-components/src/components/ContentPreview/index.vue (hunk 3 above)
Comment (ikovac) [reply packages/core-components/src/components/ContentPreview/index.vue:48]: Renato told me that you fixed this bug on VPL but was not propagated on base tailor 😅

File: package.json (hunk 2 above)
Comment (ikovac) [reply package.json:4]: sort package.json script 😉

File: .circleci/config.yml (hunk 1 above)
Comment (ikovac) [reply .circleci/config.yml:47]: yes 😫 Sometimes, I get the same issue locally than I rerun `npm install` and everything works...

File: package.json (hunk 2 above)
Comment (underscope) [reply package.json:4]: I got that part 🤡

**Pull request merged**